src/uv.c

```diff
@@ -46,7 +46,7 @@ static int uvInit(struct raft_io *io, raft_id id, const char *address)
         return rv;
     }
 
-    /* Proble file system capabilities */
+    /* Problem file system capabilities */
```

Comment:
@jsoref this looks likes it was intended to be `probe` (`UvFsProbeCapabilities`)

Reply:
Thanks @bl-ue